_episodes/documentation.md

```diff
@@ -1,33 +1,92 @@
+that find your data will not have to search for it elsewhere. Most data
+repositories have a 'description' field, which is a suitable place to provide
+any information that is not covered by the other metadata fields.
 
-Project info is typically written in the data description fields, but may also include pointers to external documents, such as the data management plans.
+> ## Grouping related files with Zenodo communities
+> Data documentation in your project may be scattered over many files of
+> different types. To bundle this information, Zenodo provides 'communities'.
+> You can upload all your files to Zenodo and add them to a community. You can
+> also track versions. Each file gets its own DOI. So you could e.g. cite your
+> data management plan in the description field of your dataset.
+{: .callout}
 
-Point forward to next episodes (e.g. formats and metadata help to document data).
+## What is the difference between metadata and documentation?
 
-Data management plan describes the planning/methods of data collection activities, data processing, analysis and (long-term) preservation.
+Following <https://howtofair.dk>, we discuss data documentation and metadata
+separately. However, the distinction between the two is not always clear. In
+this episode, we will focus on the rich, descriptive kind of documentation
+that is mostly relevant for human interpretation. In the next episodes, we will
+focus on more formalized metadata that is also, or even mainly, intended for
+machine readability.
 
-You can also log other important decisions that impact your data (sort of labjournal?!).
+## Exercises
 
-Documentation can take many forms: data management plan, papers published, (jupyter) notebooks, provencance information, etc. It is a good idea to bundle these with the data when you publish the data, so that people that find your data will not have to search for it elsewhere
+> ## Checking data documentation
+>
+> Visit a data repository of your choosing (e.g. hydroshare, 4TU, Zenodo, ...).
+> Select one or more datasets that you find interesting, and answer the
+> following questions:
+>
+> - Is the title of the dataset entry in the repository clear and informative?
+> - Is there a general description of the data?
+> - Is there a reference to some external documentation of the data (e.g. a
+>   journal publication)?
+> - Would you be able to reproduce this data, based on the provided information?
+> - Would you be able to use this dataset in your own (hypothetical) research
+>   project?
+> - Are contact details provided that you can use in case you have questions?
+> - Is there any description of the context in which this dataset was created?
+> - What could be improved about the documentation of this dataset?
+{: .challenge}
 
-{% include links.md %}
+> ## Writing your own data documentation
+>
+> For the real use case you selected in the previous episode, write a general
```

Comment:
Perhaps we need to scope this somehow. "write a single page general description"?

Reply:
Added below.